src/dir.targets

```diff
@@ -103,7 +103,7 @@
     <StartWorkingDirectory Condition="'$(StartWorkingDirectory)' == ''">$(TestPath)$(DebugTestFrameworkFolder)</StartWorkingDirectory>
     <StartAction Condition="'$(StartAction)' == ''">Program</StartAction>
     <StartProgram Condition="'$(StartProgram)' == ''">$(StartWorkingDirectory)\$(TestHost)</StartProgram>
-    <StartArguments Condition="'$(StartArguments)' == ''">$(TestCommandLine)</StartArguments>
+    <StartArguments Condition="'$(StartArguments)' == ''">$(TestCommandLine) -wait</StartArguments>
```

Comment:
This was the only conflict and is the result of merging with https://github.com/dotnet/corefx/commit/0ce28fa5464062bcba5efd4826bdd7450b2c923c#diff-aad6833a46296c5e45aba8de08674fa0R84